docs/source/README.md

```diff
@@ -188,16 +190,16 @@ Source parameters
 | `connectionSecret` | The name of the RabbitMQ secret |
 | `exchangeConfig` | Settings for the exchange |
 | `exchangeConfig.name` * | Name of the exchange |
-| `exchangeConfig.type` | [Exchange type](https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchanges). Can be `fanout`, `direct`, `topic`, `match` or `headers` |
-| `exchangeConfig.durable` * | Boolean |
-| `exchangeConfig.autoDelete` * | Boolean |
+| `exchangeConfig.type` (soon to be deprecated) | [Exchange type](https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchanges). Can be `fanout`, `direct`, `topic`, `match` or `headers` |
```

Comment:
I think all of these can just say Deprecated. We'll be removing it entirely for the 1.6 release